test/functional/VM_Test/src/j9vm/test/classunloading/ClassUnloadingTestParent.java

```diff
@@ -61,12 +61,10 @@ public void runTest() throws Exception {
 }
 
 protected static void thoroughGCandFinalization() {
-	System.gc();
 	System.gc();
 	System.runFinalization();
```

Comment:
Same comment as TestObject.java, do we still need to call runFinaliization() and do the additional System.gc()? According to our doc update the finalization should occur from the single System.gc() now.